src/aura/STG_CMP_System/STG_CMP_System.cmp

```diff
@@ -270,5 +275,94 @@
             </aura:set>
             </aura:if>
         </div>
+        <div class="slds-col slds-size--1-of-2">
+            <aura:if isTrue="{!v.namespacePrefix == 'hed__'}">
+                <ui:outputText value="{!$Label.hed.automaticHHNaming}" />
+                <aura:set attribute="else">
+                    <ui:outputText value="{!$Label.c.automaticHHNaming}" />
+                </aura:set>
+            </aura:if>
+        </div>
+        <div class="slds-col slds-size--1-of-2">
+            <div class="slds-form-element">
+                <div class="slds-form-element__control">
+                    <label class="slds-checkbox">
+                        <aura:if isTrue="{!v.isView}">
+                            <ui:outputCheckbox value="{!v.hierarchySettings.Automatic_Household_Naming__c}" class="automatic-hh-acc" />
+                            <aura:set attribute="else">
+                                <ui:inputCheckbox aura:id="automaticHHAcc" value="{!v.hierarchySettings.Automatic_Household_Naming__c}"
+                                                  class="automatic-hh-acc" />
+                                <span class="slds-checkbox--faux"></span>
+                                <span class="slds-form-element__label"></span>
+                            </aura:set>
+                        </aura:if>
+                    </label>
+                </div>
+            </div>
+        </div>
+        <div class="slds-col slds-size--1-of-1 slds-m-bottom--medium">
+            <aura:if isTrue="{!v.namespacePrefix == 'hed__'}">
+                <ui:outputText value="{!$Label.hed.automaticHHNamingHelpText}" class="slds-text-body--small" />
+                <aura:set attribute="else">
+                    <ui:outputText value="{!$Label.c.automaticHHNamingHelpText}" class="slds-text-body--small" />
+                </aura:set>
+            </aura:if>
+        </div>
+
+        <div class="slds-col slds-size--1-of-2">
+            <aura:if isTrue="{!v.namespacePrefix == 'hed__'}">
+                <ui:outputText value="{!$Label.hed.adminAccNameFormat}"/>
+            <aura:set attribute="else">
+                <ui:outputText value="{!$Label.c.adminAccNameFormat}"/>
+            </aura:set>
+            </aura:if>
+        </div>
+        
+        <div class="slds-col slds-size--1-of-2" >
```

Comment:
I think there is a small UI issue here. System does not show the current setting. 

Steps to reproduce:
1. Deploy Master branch to the org
2. Deploy the Account Naming branch to the org
3. Go to Naming Setting, click Edit and click Save
4. The Naming settings in the UI shows blank. But the value actually saved in the custom settings

Reply:
Sorry which setting are you referring to? I can't seem to reproduce it.